lib/setup.js

```diff
@@ -42,23 +43,36 @@ import config from '../config'
 const {
   API_REQUEST_TIMEOUT,
   API_URL,
+  APP_TIMEOUT,
   COOKIE_DOMAIN,
   DEFAULT_CACHE_TIME,
   IP_BLACKLIST,
   NODE_ENV,
   OPENREDIS_URL,
   REQUEST_EXPIRE_MS,
   REQUEST_LIMIT,
+  SENTRY_PRIVATE_DSN_STAGING,
+  SENTRY_PRIVATE_DSN_PRODUCTION,
   SESSION_COOKIE_KEY,
   SESSION_COOKIE_MAX_AGE,
-  SESSION_SECRET,
-  APP_TIMEOUT
+  SESSION_SECRET
 } = config
 
 export default function (app) {
   // Timeout middleware
   if (NODE_ENV === 'production') app.use(timeout(APP_TIMEOUT || '29s'))
 
+  // Error reporting
+  let ravenDSN
```

Comment:
Ternary operator for code ⛳️ ? 

````javascript
const ravenDSN = NODE_ENV === 'staging' ? SENTRY_PRIVATE_DSN_STAGING : SENTRY_PRIVATE_DSN_PRODUCTION
````

I guess those config names are long enough that line gets kinda strange.